src/vsr/clock.zig

```diff
@@ -612,8 +614,8 @@ pub fn ClockType(comptime Time: type) type {
 
 const testing = std.testing;
 const OffsetType = @import("../testing/time.zig").OffsetType;
-const DeterministicTime = @import("../testing/time.zig").Time;
-const DeterministicClock = ClockType(DeterministicTime);
+const DeterministicTime = @import("../testing/time.zig").TimeSim;
+const DeterministicClock = ClockType();
```

Comment:
I think we no longer need these aliases. Can remove DeterministicClock, and use `TimeSim` instead of `DeterministicTime`.